data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	Once confirming that you and Anna will not be heard, you begin explaining the Coalition to Anna, going in-depth about their history and their culture, and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them.`
+			`	"Will they really like my books?"`
+			choice
+				`	"They will, hopefully."`
+				`	"Well, it's better to try than to just give up."`
+			`	"I hope they do. Is there anything more you'd like to tell me about the Coalition?"`
+			choice
+				`	"By any chance, are you afraid of large beetles and spiders?"`
+			`	"No, not at all. I'm not afraid of them. Why do you ask?"`
+			choice
+				`	"Because the Coalition are a union of three alien species, who look like spiders, beetles, and centaurs.`
+			`	"Interesting. Their stories might just inspire my next novel. But, they wouldn't speak our language. How are they going to read my books?"`
+			choice
+				`	"They have a translation device."`
+				`	"I'm sure they can learn the language."`
+			`	"That's reassuring... but, how am I even going to get there?"`
+			choice
+				`	"I'll take you there."`
+			`	"Do I have to pay anything? As a struggling author, I can't afford off-world transport at all, let alone to a completely different alien race!"`
+			choice
+				`	"It's okay. I'm willing to make an exception for you."`
+			`	"Thank you so much! Just wait for a while, I need to finish my book signing event. It should be roughly 45 minutes before it's over, and then we can go to whatever this Coalition thing is," says Anna.`
+			`	You reply, "Good luck."`
+			`	As you wait for Anna, you look over your maps of Coalition space to determine the most suitable place for Anna. After much deliberation and thought, you decide on <destination>.`
+			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, as you lead her to her bunk and inform her of the destination. You prepare to leave the spaceport.`
+				accept
+	on complete
+		conversation
+			`	You leave the ship with Anna following you. The moment you are out of your ship, three Kimek surround you and Anna. The Kimek in the middle speaks, and then brief silence. The silence is broken as their translation box speaks.`
+			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`
+			`	You and Anna introduce yourself. Anna's introduction is longer, while you only need to state your name, seeing as you are more familiar with the Coalition now.`
+			`	"About what, do you write?", asks the Kimek.`
+			`	"Oh, I write science fiction books," says Anna.`
+			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`
```

Comment:
It's not so much that they switch to your language, I'd phrase it more as them finally giving the translator box a chance to translate (maybe they switched it off to discuss among themselves?).